Modules/FindCUDA.cmake

```diff
@@ -634,6 +682,9 @@ find_program(CUDA_NVCC_EXECUTABLE
 find_program(CUDA_NVCC_EXECUTABLE nvcc)
 mark_as_advanced(CUDA_NVCC_EXECUTABLE)
 
+#MESSAGE("CUDA_NVCC_EXECUTABLE: ${CUDA_NVCC_EXECUTABLE} CUDA_TOOLKIT_ROOT_DIR: ${CUDA_TOOLKIT_ROOT_DIR} CUDA_TOOLKIT_TARGET_DIR: ${CUDA_TOOLKIT_TARGET_DIR}")
```

Comment:
These messages only print during configure time, and don't follow any "verbose" mode that I know of (perhaps there is a new verbose mode).

Really this sort of debugging output belongs in the calling code and not in here.